global.d.ts

```diff
@@ -89,6 +92,11 @@ interface TRun {
   workspaceId: TId;
```

Comment:
WorkSpace Id to be converted to project name, or?

Reply:
For now, workSpaceId is used in many places like old pipelines and runs that's why we need it as a type in typescript.